src/axolotl/prompt_tokenizers.py

```diff
@@ -422,38 +425,6 @@ def tokenize_prompt(self, prompt):
         except (KeyError, AssertionError, IndexError) as err:
             raise InvalidDataException(str(err)) from err
 
-    def _tokenize(self, prompt, add_eos_token=True, strip_bos_token=False):
-        if not prompt.strip():
-            LOG.warning("Empty text requested for tokenization.")
-            result = BatchEncoding(data={"input_ids": [], "attention_mask": []})
-        else:
-            result = self.tokenizer(
-                prompt,
-                truncation=True,
-                max_length=self.sequence_len,
```

Comment:
Please note that it was `sequence_len` before but the merged `_tokenize` method will use `max_length`. They were different.